kubejs/startup_scripts/gtceu/apiary.js

```diff
@@ -58,9 +57,7 @@ GTCEuStartupEvents.registry('gtceu:machine', event => {
         .recipeType('apiary_ii')
         .appearanceBlock(GTBlocks.CASING_TITANIUM_STABLE)
         .tooltips(Component.translatable("kubejs.apiary_ii.tooltip.bee_requirements"))
-        // .recipeModifier(GTRecipeModifiers.PARALLEL_HATCH.apply(OverclockingLogic.NON_PERFECT_OVERCLOCK, GTRecipeModifiers.ELECTRIC_OVERCLOCK))
-        // .recipeModifiers([GTRecipeModifiers.PARALLEL_HATCH, GTRecipeModifiers.ELECTRIC_OVERCLOCK.apply(OverclockingLogic.NON_PERFECT_OVERCLOCK)])
-        //.recipeModifiers(GTRecipeModifiers.ELECTRIC_OVERCLOCK.apply(OverclockingLogic.NON_PERFECT_OVERCLOCK))
+        .recipeModifiers(GTRecipeModifiers.ELECTRIC_OVERCLOCK.apply(OverclockingLogic.NON_PERFECT_OVERCLOCK))
```

Comment:
Pretty sure this line is missing the [ ]. You would get startup error without it.

Reply:
I can add it, but I believe that startup error only occurs when there is more than one argument. In this case we just have the one argument. Perhaps it should just be `.recipeModifier` though rather than `.recipeModifiers`, I'll double check it - thanks!